### PR TITLE
added symbolic link to setupscript

### PR DESCRIPTION
**Pull request by Justin-Hung**

### CODE REVIEW

**Comment by chordatgh1234** [comment]
it says my comment cant be blank.

**Comment by bill-luu** [comment]
Looks good, I'll verify it later. 

In the future, adding "Fixes #13" to the description will link this PR to the issue, so that when this is closed, the issue gets closed as well!

File: SetupScript.sh [hunk 1]
```diff
@@ -125,7 +125,7 @@ mv /opt/Epsilon-Raspberry/QT_CFLAGS_DBUS.patch /home/pi/qt5
 	-prefix /home/pi/qt5 |& tee "output_configure.txt")
 (cd /home/pi/qt5/qtbase && make |& tee "output_make.txt")
 (cd /home/pi/qt5/qtbase && make install |& tee "output_make_install.txt")
-qtchooser -install qt55 /home/pi/qt5/bin/qmake
+ln -s /home/pi/qt5/bin/qmake /usr/bin/qmake
```
Comment (bill-luu) [inline SetupScript.sh]: I think you have to remove the old link before doing this. 

`rm /usr/bin/qmake`

**Review by bill-luu — CHANGES_REQUESTED** [review]

File: SetupScript.sh (hunk 1 above)
Comment (ExtraGravity) [reply SetupScript.sh]: In general it's good practice to not remove stuff in scripts that the script did not make itself. Making a backup is better.

`mv /usr/bin/qmake /usr/bin/qmake.bak`

File: SetupScript.sh [hunk 2]
```diff
@@ -125,6 +125,8 @@ mv /opt/Epsilon-Raspberry/QT_CFLAGS_DBUS.patch /home/pi/qt5
 	-prefix /home/pi/qt5 |& tee "output_configure.txt")
 (cd /home/pi/qt5/qtbase && make |& tee "output_make.txt")
 (cd /home/pi/qt5/qtbase && make install |& tee "output_make_install.txt")
+mv /usr/bin/qmake /usr/bin/qmake.bak
+rm /usr/bin/qmake
```
Comment (bill-luu) [inline SetupScript.sh]: Don't need this now, since qmake got renamed to qmake.bak, this rm command won't do anything.

File: qmake [hunk 3]
```diff
@@ -0,0 +1 @@
+/home/eimoo/Qt/5.5/gcc_64/bin/qmake
```
Comment (ExtraGravity) [inline qmake]: This path will only work on one machine

Comment (bill-luu) [reply qmake]: lol I didn't even see this file, remove it!

**Review by bill-luu — APPROVED** [review]

**Pull request merged**